org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/grid/WQuickEntry.java

```diff
@@ -141,15 +144,15 @@ public WQuickEntry(int AD_Window_ID)
 		m_AD_Window_ID = AD_Window_ID;
 		parent_WindowNo = 0;
 		m_WindowNo = 0;
-		log.info("R/O=" + m_readOnly);
+		log.info("R/O=" + isReadOnly());
 	}	//	WQuickEntry
 
 	/**
 	 *	Static Init
 	 * 	@throws Exception
 	 */
 
-	private void jbInit() throws Exception
+	protected void jbInit() throws Exception
```

Comment:
would be better to rename this to just init(), jbinit sounds weird.

Reply:
:)

Reply:
if that's the case, perhaps there isn't a needs here to change it to protected ? otherwise, you can leave it as it is.

Reply:
It is protected to allow for further reaching layout customization in the child class.
I would rather not make it private because of this.

Reply:
alright, better rename it to something that wouldn't cause compatibility issue, for e.g initQuickEntryLayout(). There might be subclass of WQuickEntry out there that have the same jbInit private method.

Reply:
WPreference extends WQuickEntry and have its own protected jbInit method. Therefore, if you change WQuickEntry's jbInit method to protected, you MUST rename it to something else, otherwise it will break WPreference.

Reply:
You are right. I again forgot to consider the chance that some unpublished customization could break as a consequence. Will rename the method to initLayout() and make it protected.

Reply:
Done.